src/sentry/static/sentry/app/components/smartSearchBar.jsx

```diff
@@ -614,11 +658,21 @@ class SmartSearchBar extends React.Component {
             />
             <span className="icon-search" />
             {this.state.query !== '' && (
-              <div>
+              <React.Fragment>
+                {this.props.hasPinnedSearch && (
+                  <PinButton
+                    isPinned={!!this.props.pinnedSearch}
```

Comment:
`pinnedSearch` currently will never be set - this won't be fully functional until we integrate with new UI.